Formula/a/apprise.rb

```diff
@@ -92,7 +92,7 @@ def my_wrapper(body, title, *args, **kwargs):
     assert_match \
       "#{brewtest_notification_title}: #{brewtest_notification_body}", \
       shell_output(
-        "#{bin}/apprise" \
+        bin/"apprise" \
         + " " + %Q(-P "#{brewtest_notifier_file}") \
         + " " + %Q(-t "#{brewtest_notification_title}") \
         + " " + %Q(-b "#{brewtest_notification_body}") \
```

Comment:
Well this is some interesting syntax!

Reply:
Have literally never seen this before. There's always some edge case 🤦‍♂️

Reply:
```diff
diff --git a/Formula/a/apprise.rb b/Formula/a/apprise.rb
index 608384cfbb3..8d9e659931d 100644
--- a/Formula/a/apprise.rb
+++ b/Formula/a/apprise.rb
@@ -72,7 +72,7 @@ class Apprise < Formula
 
   test do
     # Setup a custom notifier that can be passed in as a plugin
-    brewtest_notifier_file = "#{testpath}/brewtest_notifier.py"
+    file = "#{testpath}/brewtest_notifier.py"
     apprise_plugin_definition = <<~EOS
       from apprise.decorators import notify
 
@@ -82,22 +82,14 @@ class Apprise < Formula
         print("{}: {}".format(title, body))
     EOS
 
-    File.write(brewtest_notifier_file, apprise_plugin_definition)
+    File.write(file, apprise_plugin_definition)
 
     charset = Array("A".."Z") + Array("a".."z") + Array(0..9)
-    brewtest_notification_title = charset.sample(32).join
-    brewtest_notification_body = charset.sample(256).join
+    title = charset.sample(32).join
+    body = charset.sample(256).join
 
     # Run the custom notifier and make sure the output matches the expected value
-    assert_match \
-      "#{brewtest_notification_title}: #{brewtest_notification_body}", \
-      shell_output(
-        "#{bin}/apprise" \
-        + " " + %Q(-P "#{brewtest_notifier_file}") \
-        + " " + %Q(-t "#{brewtest_notification_title}") \
-        + " " + %Q(-b "#{brewtest_notification_body}") \
-        + " " + '"brewtest://"' \
-        + " " + "2>&1",
-      )
+    assert_match "#{title}: #{body}",
+      shell_output("#{bin}/apprise -P \"#{file}\" -t \"#{title}\" -b \"#{body}\" \"brewtest://\" 2>&1")
   end
 end
```

This makes it more normal, if you wanted to do that in a separate PR.